.circleci/config.yml

```diff
@@ -7,6 +7,15 @@ defaults: &defaults
   working_directory: /home/circleci/project
   docker:
     - image: circleci/node:10.15.3
+      environment:
```

Comment:
Why are you adding all of these to the `defaults`?

Reply:
Just to get it working. This is still in draft.